client/src/main/java/org/apache/nemo/client/ClientUtils.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.nemo.client;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.nemo.common.Util;
+import org.apache.nemo.common.exception.MetricException;
+import org.apache.nemo.runtime.common.comm.ControlMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+
+/**
+ * Utility class for the Client.
+ */
+public final class ClientUtils {
+  private static final Logger LOG = LoggerFactory.getLogger(ClientUtils.class.getName());
+
+  /**
+   * Private constructor.
+   */
+  private ClientUtils() {
+  }
+
+  /**
+   * Handler for the launch optimization message.
+   *
+   * @param message the message received from the driver.
+   */
+  static void launchOptimizationHandler(final ControlMessage.DriverToClientMessage message,
+                                        final DriverRPCServer driverRPCServer) {
+    switch (message.getOptimizationType()) {
+      case "xgboost":
```

Comment:
It would be better to use enum instead of raw string

Reply:
🆗